src/PhappCommandBase.php

```diff
@@ -84,6 +84,8 @@ protected function initShellEnvironment() {
    *   Thrown when the environment is undefined.
    */
   protected function getPhappEnviromentVariables($directory = './') {
+    // Get the current's directory manifest.
+    $manifest = PhappManifest::getInstance($directory);
```

Comment:
This will reparse the manifest, while it should be already in the calling code. Could that be unified so save re-parsing it?